frontend/src/TableComponents/SortMenu.jsx

```diff
@@ -0,0 +1,143 @@
+import { Button, Menu, MenuItem, Divider } from "@mui/material";
+import { useState } from "react";
+import "./SortMenu.css";
+
+function SortMenu({ setSortType }) {
+  const [anchorEl, setAnchorEl] = useState(null);
+  const open = Boolean(anchorEl);
```

Comment:
- imo instead of `Boolean(anchorEl)` it would be more readable to say `anchorEl == null`
- boolean naming converntion typically begins with `is` or `has` so `isOpen`